tests/periph_timer/main.c

```diff
@@ -34,15 +34,17 @@
 #define MAX_CHANNELS        (10U)
 #define TIM_SPEED           (1000000ul) /* try to run with 1MHz */
 #define CHAN_OFFSET         (5000U)     /* fire every 5ms */
+#define COOKIE              (100U)      /* for checking if arg is passed */
 
 static volatile int fired;
 static volatile uint32_t sw_count;
 static volatile uint32_t timeouts[MAX_CHANNELS];
+static volatile int args[MAX_CHANNELS];
```

Comment:
this needs to be `unsinged` (or you have to do some casting below), otherwise gcc complains..

Reply:
Changed it to uint32_t, and the initial value to UINT32_MAX.